frontend/src/pages/moments/momentsCreateNewView.tsx

```diff
@@ -1,17 +1,152 @@
-import { IonContent, IonPage } from '@ionic/react'
-import { Header } from '../../components'
+import {
+  IonContent,
+  IonPage,
+  IonImg,
+  IonItem,
+  IonInput,
+  IonLabel,
+  IonList,
+  IonSelect,
+  IonSelectOption,
+  IonButton,
+  IonTextarea,
+  IonToast,
+} from '@ionic/react'
 import { RouteComponentProps } from 'react-router'
+import { Dispatch, SetStateAction, useState } from 'react'
+import { AudioRecorder } from '../../components/media/audioRecorder'
+import { VideoRecorder } from '../../components/media/videoRecorder'
+import { ImageRecorder } from '../../components/media/imageRecorder'
+import { LargeHeader, Header } from '../../components/header'
+import { TextRecorder } from '../../components/media/TextRecorder'
+import { createMomentAPI, Media } from '../../api/moment/createMoment'
 
-interface Props
-  extends RouteComponentProps<{
-    id: string
-  }> {}
+export interface Moment {
+  title: string
+  tags: string[]
+}
+interface Props extends RouteComponentProps<{}> {}
 
 export const MomentsCreateNewView: React.FC<Props> = ({ match, history }) => {
+  const [isToastPresent, setIsToastPresent] = useState(false)
+  const [media, setMedia]: [Media, Dispatch<SetStateAction<Media>>] = useState({
+    type: '',
+    data: '',
+    name: '',
+  })
```

Comment:
Not the correct place for the comment but:

We use `any` in all child elements that receive a reference to `setMedia` for the type that it manipulate. Couldn't we also use types there?